sky/packages/sky/lib/src/rendering/layer.dart

```diff
@@ -87,6 +87,27 @@ class PictureLayer extends Layer {
 
 }
 
+/// A layer that indicates to the compositor that it should display
+/// certain statistics within it
+class StatisticsLayer extends Layer {
+  StatisticsLayer(
+      { Offset offset: Offset.zero, this.paintBounds, this.optionsMask })
```

Comment:
We usually put the { on the previous line, then indent each argument 2 spaces from the constructor, one per line, then have the `}) : super(...)` on its own line.